packages/twenty-front/src/modules/ui/layout/dropdown/components/Dropdown.tsx

```diff
@@ -25,19 +25,24 @@ import { isDefined } from 'twenty-shared/utils';
 import { useIsMobile } from 'twenty-ui/utilities';
 import { useDropdown } from '../hooks/useDropdown';
 
+type Width = `${string}px` | `${number}%` | 'auto' | number;
 const StyledDropdownFallbackAnchor = styled.div`
   left: 0;
   position: fixed;
   top: 0;
 `;
 
-const StyledClickableComponent = styled.div`
+const StyledClickableComponent = styled.div<{
+  width?: Width;
+}>`
   height: fit-content;
+  width: ${({ width }) => width ?? 'auto'};
 `;
 
 export type DropdownProps = {
   className?: string;
   clickableComponent?: ReactNode;
+  clickableComponentWidth?: Width;
```

Comment:
I know you took the time to find the best solution to this problem, but I wonder if we didn't find an API design flaw in the Dropdown component. We can provide a `clickableComponent`, but won't the width of the clickable area expand to the bounds of the `clickableComponent`? This looks strange to me, and maybe we should discuss it.